src/main/java/org/fundacion/jala/converter/models/UserSQL.java

```diff
@@ -65,9 +69,9 @@ public static void editUserData(final int userId, final String userName, final S
     }
 
     /**
-     * Deletes a user.
+     * Deletes a user of the database.
```

Comment:
It would be better if we use "from" instead of "of"."

Reply:
ok